mirdata/io.py

```diff
@@ -4,6 +4,7 @@
 
 import numpy as np
 import pretty_midi
+from smart_open import open
```

Comment:
I think you can delete this line

Reply:
I think it'd needed - we call the system `open` in the decorator, and we'll need to overwrite it with smart_open to support remote files, no?

Reply:
Ah forgot that was in this file. nvm then.